core/modules/layout/layout.theme.inc

```diff
@@ -202,14 +202,15 @@ function theme_layout_settings_context_table($variables) {
   $header = array(
     t('Context'),
     t('Type'),
+    t('Operations'),
   );
   $rows = array();
   foreach (element_children($element) as $key) {
     $argument_parts = $parts;
     $row = array();
     if (is_numeric($key)) {
-      if (isset($element[$key]['settings']['#value']['entity'])) {
-        $row[] = t('Special context: @id', array('@id' => $element[$key]['settings']['#value']['entity']));
+      if (isset($element[$key]['context']['#value']->settings['entity'])) {
+        $row[] = t('Special context: @id', array('@id' => $element[$key]['context']['#value']->settings['entity']));
```

Comment:
We should make sure that whatever we pass to `@id` exists. I got this in the PR sandbox:

![Screen Shot 2020-04-28 at 12 47 15 pm](https://user-images.githubusercontent.com/2423362/80473154-68fbac80-894e-11ea-85d8-8495c73e9818.png)